server/services/converters_test.go

```diff
@@ -33,3 +33,33 @@ func TestBlockIdentifierToAccountQueryOptions(t *testing.T) {
 		require.ErrorContains(t, err, "encoding/hex: invalid byte")
 	})
 }
+
+func TestUtf8ToHex(t *testing.T) {
+	require.Equal(t, "68656c6c6f", utf8ToHex("hello"))
+	require.Equal(t, "776f726c64", utf8ToHex("world"))
```

Comment:
Add a test with an odd num of hex chars.

Reply:
Added.